server/bcoinRouter.js

```diff
@@ -7,15 +7,22 @@ const routerWithClient = client => {
   const bcoinRouter = express.Router({ mergeParams: true });
 
   bcoinRouter.all('*', async (req, res, next) => {
-    const { method, path, body } = req;
+    const { method, path, body, query } = req;
+    // use query params for GET request, otherwise use body
+    const payload = method === 'GET' ? query : body;
     try {
       logger.debug(
-        `client: ${client.constructor.name}, method: ${method}, path: ${path}`
+        `client: ${client.constructor.name}, method: ${method},`,
+        // eslint-disable-next-line
+        `path: ${path}, query:`,
+        query,
+        ', body:',
+        body
       );
-      const bcoinResponse = await client.request(method, path, body);
-      logger.debug('server response:', bcoinResponse ? bcoinResponse : 'null');
-      if (bcoinResponse) return res.status(200).json(bcoinResponse);
-      // when bcoinResponse is null due to
```

Comment:
Changed it from `bcoinResponse` to `response` because this method just sends a request and doesn't need to know who it is sending it to